### PR TITLE
Update traces to metrics doc

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What is this feature?**

Update the trace to metrics doc to add more details and expand the procedure. 

Related: https://github.com/grafana/website/pull/20982

**Why do we need this feature?**

Content was slightly out of date. 

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

Fixes #

**Special notes for your reviewer:**

Please check that:
- [X] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/datasources/tempo/configure-tempo-data-source.md [hunk 1]
```diff
@@ -100,8 +100,9 @@ There are two ways to configure the trace to logs feature:
...
-1. Optionally switch on the **Filter by trace ID** and/or **Filter by span ID** setting to further filter the logs if your logs consistently contain trace or span IDs.
+1. Select which tags to use in the logs query.
+   The tags you configure must be present in the span's attributes or resources for a trace to logs span link to appear. You can optionally configure a new name for the tag. This is useful, for example, if the tag has dots in the name and the target data source does not allow using dots in labels. In that case, you can for example remap `http.status` (the span attribute) to `http_status` (the data source field). "Data source" in this context can refer to Loki, or another log data source.
+1. Optional: Switch on the **Filter by trace ID** and/or **Filter by span ID** setting to further filter the logs if your logs consistently contain trace or span IDs.
```
Comment (jdbaldry) [inline docs/sources/datasources/tempo/configure-tempo-data-source.md:105]: https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanaandor

```suggestion
1. Optional: Switch on the **Filter by trace ID** and **Filter by span ID** settings to further filter the logs if your logs consistently contain trace or span IDs.
```

File: docs/sources/datasources/tempo/configure-tempo-data-source.md [hunk 2]
```diff
@@ -141,19 +142,33 @@ There are two ways to configure the trace to metrics feature:
 - Use a basic configuration with a default query, or
 - Configure one or more custom queries where you can use a [template language](ref:variable-syntax) to interpolate variables from the trace or span.
 
-### Simple config
+Refer to the Trace to metrics configuration options section to learn about the available options.
```
Comment (jdbaldry) [inline docs/sources/datasources/tempo/configure-tempo-data-source.md:145]: Should this be capitalized?

Is it UI thing that should be bold?

File: docs/sources/datasources/tempo/configure-tempo-data-source.md [hunk 3]
```diff
@@ -141,19 +142,33 @@ There are two ways to configure the trace to metrics feature:
...
+
+### Set up a simple configuration
 
 To use a simple configuration, follow these steps:
 
 1. Select a metrics data source from the **Data source** drop-down.
+1. Optional: Change the **Span start time shift** and/or **Span end time shift**. The default start time shift is -2 minutes and 2 minutes for end time shift.
```
Comment (jdbaldry) [inline docs/sources/datasources/tempo/configure-tempo-data-source.md:152]: https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanaandor

```suggestion
1. Optional: Change the **Span start time shift** and **Span end time shift**. The default start time shift is -2 minutes and 2 minutes for end time shift.
```

File: docs/sources/datasources/tempo/configure-tempo-data-source.md (hunk 2 above)
Comment (jdbaldry) [reply docs/sources/datasources/tempo/configure-tempo-data-source.md:145]: Oh and perhaps it should be a link to the later heading?

File: docs/sources/datasources/tempo/configure-tempo-data-source.md (hunk 1 above)
Comment (knylander-grafana) [reply docs/sources/datasources/tempo/configure-tempo-data-source.md:105]: Okay, but. I have some disagreement with the linting rule on this. In procedures, using "and" has an implicit instruction to select all options listed. Using "and/or", while not pretty, is a succinct way of saying "You can use either or both of these options:". I need to go back into the Traces to logs part to verify the user's action to see how to rewrite this sentence. I am not sure changing it to "and" is correct.

File: docs/sources/datasources/tempo/configure-tempo-data-source.md (hunk 3 above)
Comment (knylander-grafana) [reply docs/sources/datasources/tempo/configure-tempo-data-source.md:152]: Same as before. In this case, it's an option where someone could change one or both. I'm not sure if someone would want to have them as different numbers, but technically they could. I'll see if there is a clearer way to message this and will check with a dev to see if both of these have to use the same value (i.e., -3/+3) or if different values (i.e., -3/+5) are valid.

Comment (jdbaldry) [reply docs/sources/datasources/tempo/configure-tempo-data-source.md:152]: Perhaps it's most clear to have them as two optional steps where each is independently described?

Comment (jdbaldry) [reply docs/sources/datasources/tempo/configure-tempo-data-source.md:152]: I believe the intention of the style rule is that being explicit is preferable to being succinct.

Comment (knylander-grafana) [reply docs/sources/datasources/tempo/configure-tempo-data-source.md:152]: I can do that. Thank you for the suggestion.

**Comment by knylander-grafana** [comment]
@jdbaldry Addressed the issues. Would you re-review?

**Review by jdbaldry — APPROVED** [review]
Lovely

**Pull request merged**